### PR TITLE
output_transform 

### PR DESCRIPTION
**Pull request by AntonioMirarchi**
We can enhance the initialization of the external_module by adding the `output_transform` argument. In the torchmd-net.calculator, the default value of this argument is set to None, which means it returns the identity transformation. However, now you have the flexibility to specify the desired transformer to be applied through the external arguments in the YAML configuration file of torchmd. Keep in mind that torchmd operates using kcal/mol for energy and kcal/mol/A for forces as units of measure.
Change according to torchm-net PR [#167](https://github.com/torchmd/torchmd-net/pull/167)

### CODE REVIEW

**Review by RaulPPelaez — APPROVED** [review]
Looks good!

File: torchmd/run.py [hunk 1]
```diff
@@ -10,13 +10,12 @@
 import numpy as np
 from tqdm import tqdm
 import argparse
-import math
 import importlib
 from torchmd.integrator import maxwell_boltzmann
 from torchmd.utils import save_argparse, LogWriter, LoadFromFile
 from torchmd.minimizers import minimize_bfgs
 from npzMol import npzMolecule
-from utils import elements, converter_xyz_output
```
[the remark this answers is not in the document]
Comment (AntonioMirarchi) [reply torchmd/run.py:19]: I already removed it

[the remark this answers is not in the document]
Comment (RaulPPelaez) [reply torchmd/run.py:19]: ups sorry, must have miss read.

**Comment by RaulPPelaez** [comment]
cc @stefdoerr . I cannot merge on torchmd.

**Comment by stefdoerr** [comment]
Can you approve though?

**Review by RaulPPelaez — APPROVED** [review]

**Pull request merged**